content/root/glossary.textile

```diff
@@ -0,0 +1,212 @@
+
+Access your Ably dashboard by "logging in":https://ably.com/login or "signing up":https://ably.com/sign-up.     
+
+h2(#dead-letter). Dead letter queue
+
+Dead letter queues store messages that were intended to be delivered to a queue, but have failed to be processed correctly. They are created when a queue is provisioned, with one dead letter queue per application, not per queue.
+
+See the "dead letter queue documentation":/general/queues#deadletter for further information.
+
+h2(#deltas). Deltas
+
+Deltas (or delta compression) are a feature that enable a subscriber to only receive the difference between the payload of a message and its predecessor. They are used when there is a degree of similarity between successive messages.
+
+See the "deltas documentation":/realtime/channels/channel-parameters/deltas for further information.
+
+h2(#firehose). Firehose
+
+Reactor Firehose is a feature that allows the realtime data published in Ably to be streamed directly into another streaming or queuing service such as Amazon Kinesis or Kafka.
+
+See the "Firehose documentation":/general/firehose for further information.
+
+h2(#idempotent). Idempotent
+
+An idempotent operation is one whose effect is the same whether it is performed once or multiple times. In the context of Ably, idempotence ensures that retried publish attempts, such as in response to a network failure, do not result in duplicate messages being published.
+
+See the "idempotent messages documentation":/rest/messages#idempotent for further information.
+
+h2(#inband-occupancy). Inband channel occupancy
+
+The inband channel occupancy feature allows a client to subscribe to occupancy events related to a channel. The events are then delivered to that client as messages on the channel.
+
+See the "inband channel occupancy documentation":/realtime/inband-occupancy for further information
+
+h2(#iot). IoT
+
+Internet of Things (IoT) is a term used to describe physical objects using software or sensors that are connected to the internet. Examples include smart meters, smart appliances and smart security systems.
+
+See "Wikipedia":https://en.wikipedia.org/wiki/Internet_of_things for further information.
+
+h2(#jwt). JWT
+
+JSON Web Token (JWT) is an open standard for securely representing claims transmitted between parties. The claims and metadata of a JWT are in JSON format and can be verified by recipients because they are cryptographically signed. JWTs can be used for authenticating with Ably.
+
+See "jwt.io":https://jwt.io/ for further information.
+
+h2(#latency). Latency
+
+Latency is a measure of the delay between an operation being initiated and its effect. In the context of Ably, latency typically refers to the time it takes for a message to be published in a channel on one connection and received in the channel by another connection.
```

Comment:
Would it be clearer to talk about publishers and subsribers here, instead of connections? i.e. "latency typically refers to the time elapsed between a message being publshed on a channel and being received by a channel subscriber".